### PR TITLE
Nr 49260 quick start alerts nginx-ingress

### PR DESCRIPTION
**Pull request by gsidhwani-nr**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

## Pre merge checklist

- [X ] Did you check you NRQL syntax? - Does it work?
- [ X] Did you include an InstallPlan and Documentation reference?
- [X ] Are all documentation links publically accessible?
- [ X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [ X] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [ X] Does the PR contain a screenshot for each of your dashboards?
- [X ] Do your screenshots show data?
- [ X] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [X ] Did you check that your alerts actually work?

### CODE REVIEW

**Comment by aswanson-nr** [comment]
Hey again @gsidhwani-nr, we now have a ticket on the backlog to get validation updated to allow for these new `operator` terms. If you'd like to get this merged in before we can get that updated, swapping to a `ABOVE` or `BELOW` value should get you most of the way there!

**Comment by gsidhwani-nr** [comment]
Updated to BELOW as suggested.

File: alert-policies/nginx-ingress/Ingress-certificate-expiry-alert.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  query: "FROM Metric SELECT floor((latest(nginx_ingress_controller_ssl_expire_time_seconds) - (aggregationendtime()/1000)) / (24*60*60)) FACET host WHERE instrumentation.name = 'remote-write'"
+
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: BELOW_OR_EQUALS
```
Comment (aswanson-nr) [inline alert-policies/nginx-ingress/Ingress-certificate-expiry-alert.yml:16]: Found one more `BELOW_OR_EQUALS`!

**Comment by gsidhwani-nr** [comment]
Fixed with BELOW.

File: alert-policies/nginx-ingress/High-http-4xx-5xx-error-alert.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+name: High http 4xx/5xx error alert
+
+description: |+
+ This alert is triggered when the http request success rate drops below 95 percent. This alert may be further configured as per the desired success percentage requirements.
```
Comment (thezackm) [inline alert-policies/nginx-ingress/High-http-4xx-5xx-error-alert.yml:4]: recommend updating for clarity and updating the `HTTP` initialism:

>This alert is triggered when the HTTP request success rate drops below 95 percent. The alert threshold can be changed to your desired success percentage requirements.

File: alert-policies/nginx-ingress/Ingress-certificate-expiry-alert.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+name: Ingress certificate expiry alert
```
Comment (thezackm) [inline alert-policies/nginx-ingress/Ingress-certificate-expiry-alert.yml:1]: recc: change `expiry` to `expiration`

**Review by thezackm — CHANGES_REQUESTED** [review]
@gsidhwani-nr - added some comments on formatting changes requested as part of the `prometheus-growth-hack` initiative

@aswanson-nr - curious if you have any opinions about the note on `histogrampercentile()` vs `bucketpercentile()` in the `P99-latency-alert.yml` condition.

File: alert-policies/nginx-ingress/P99-latency-alert.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+name: P99 latency alert
+
+description: |+
+ This alert is triggered when there is a latency in processing a request that is more than 100 milliseconds for the 99th percentile of request volume. This sample alert may be further configured as per desired latency requirements.
+type: STATIC
+nrql:
+  query: "SELECT histogrampercentile(nginx_ingress_controller_request_duration_seconds_bucket, (100 * 0.99))*1000 FROM Metric"
```
[the remark this answers is not in the document]
Comment (aswanson-nr) [reply alert-policies/nginx-ingress/P99-latency-alert.yml:7]: Hey @thezackm, I'm always for avoiding deprecated functions!

**Comment by gsidhwani-nr** [comment]
Implemented feedback !

File: alert-policies/nginx-ingress/Ingress-certificate-expiry-alert.yml (hunk 1 above)
Comment (gsidhwani-nr) [reply alert-policies/nginx-ingress/Ingress-certificate-expiry-alert.yml]: Updated

File: alert-policies/nginx-ingress/High-http-4xx-5xx-error-alert.yml [hunk 5]
```diff
@@ -0,0 +1,34 @@
+name: High http 4xx/5xx error alert
```
[the remark this answers is not in the document]
Comment (gsidhwani-nr) [reply alert-policies/nginx-ingress/High-http-4xx-5xx-error-alert.yml:1]: updated

File: alert-policies/nginx-ingress/High-http-4xx-5xx-error-alert.yml [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+description: |+
+ This alert is triggered when the http request success rate drops below 95 percent. The alert threshold can be changed to your desired latency alerting requirements.
```
[the remark this answers is not in the document]
Comment (gsidhwani-nr) [reply alert-policies/nginx-ingress/High-http-4xx-5xx-error-alert.yml:4]: updated

File: alert-policies/nginx-ingress/P99-latency-alert.yml [hunk 7]
```diff
@@ -0,0 +1,26 @@
+name: P99 latency alert
+
+description: |+
+ This alert is triggered when the HTTP request success rate drops below 95 percent. The alert threshold can be changed to your desired success percentage requirements.
+type: STATIC
+nrql:
+  query: "SELECT bucketpercentile(nginx_ingress_controller_request_duration_seconds_bucket, (100 * 0.99))*1000 FROM Metric"
```
[the remark this answers is not in the document]
Comment (gsidhwani-nr) [reply alert-policies/nginx-ingress/P99-latency-alert.yml]: Updated as suggested. Also updated the description field.

File: alert-policies/nginx-ingress/High-http-4xx-5xx-error-alert.yml [hunk 8]
```diff
@@ -0,0 +1,34 @@
+name: High HTTP 4xx/5xx error alert
+
+description: |+
+ This alert is triggered when the HTTP request success rate drops below 95 percent. The alert threshold can be changed to your desired latency alerting requirements.
```
Comment (thezackm) [inline alert-policies/nginx-ingress/High-http-4xx-5xx-error-alert.yml:4]: small typo here: 
```
desired latency alerting requirements
```

this is an error alert, should be something more like:
```
desired error alerting requirements
```

**Review by thezackm — CHANGES_REQUESTED** [review]
@gsidhwani-nr - one small typo change and this one is good to go; thanks for all the work on this!

**Comment by gsidhwani-nr** [comment]
Applied the changes !

**Review by thezackm — APPROVED** [review]
approved for the `prometheus-growth` project; thanks @gsidhwani-nr!

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**